src/shell/badge-wrap.tsx

```diff
@@ -26,20 +26,37 @@ const MiniBadge = styled(Badge)`
 		font-size: 0.625rem;
 		line-height: normal;
 	}
+
+	& > div {
+		font-size: 0.625rem;
+		line-height: normal;
+	}
+`;
+
+const MiniIcon = styled(Icon)`
+	position: absolute;
+	bottom: 25%;
+	right: 25%;
+	transform: translate(30%, 30%);
+	user-select: none;
+	cursor: pointer;
+	pointer-events: none;
+	z-index: 99;
 `;
 
 const BadgeWrap: FC<{ badge: BadgeInfo }> = forwardRef<HTMLDivElement, { badge: BadgeInfo }>(
 	function BadgeWrapFn({ badge, children }, ref): JSX.Element {
 		return (
 			<Container width={'3rem'} height={'3rem'} style={{ position: 'relative' }} ref={ref}>
-				{badge.show && (
-					<MiniBadge
-						color={'gray6'}
-						backgroundColor={badge.color ?? 'primary'}
-						data-testid={'badge-counter'}
-						value={badge.showCount ? badge.count ?? 0 : ''}
-					/>
-				)}
+				{(badge.show && badge.icon && <MiniIcon icon={badge.icon} color={badge.color} />) ||
```

Comment:
The new badge supports the icon inside it, I feel it would be better to use that instead of rendering an icon

Reply:
@beawar Together with @CMazzilli  we tried to use the badge but we saw that its icon is too small. For this reason we thought it would be better to use the icon as we currently use on Files. 

We will re-evaluate the use of the badge when we begin the following issue: https://zextras.atlassian.net/browse/CDS-241